crypto/pem/pem_lib.c

```diff
@@ -828,6 +828,13 @@ static int get_header_and_data(BIO *bp, BIO **header, BIO **data, char *name,
             goto err;
         }
 
+        /* Check if line has been read completely or if only part of the line
+         * has been read. Keep the previous value to ignore newlines that
+         * appear due to reading a line up until the char before the newline
```

Comment:
If you make other changes (e.g., to the formatting of the start of the comment), please also add a period to end this sentence.